_posts/2017-11-07-haskell-learning-group-our-progress-and-updates.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Haskell Learning Group: Our Progress and Updates"
+date: 2017-11-07 00:03:30 +0200
+categories: haskell how-we-learn
+author_name: Sergii Paryzhskyi
+author_url : /author/sergii_paryzhskyi
+author_avatar: sergii_paryzhskyi
+show_avatar : true
+read_time : 7
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-17-first-haskell-meeting/haskell-poster.jpg
+---
+
+This post is a quick update and the latest news about our Haskell learning group. Even though there were no blog posts to this topic recently, we are still having weekly meetings and mostly stick to the plan. That of course implies preparation time, spend between our sessions. There is no rush in our learning process and we take time to discuss interesting topics a bit longer, especially when all participants are involved. That being said, we might have a plan to go through one chapter per session but it also fine if it is less than that.
+
+One thing we've noticed and that was to expect, is that the difficulty of a content is slowly increasing. We are spending more time reading chapters. Sometimes the same material has to be read multiple times in order to fully understand what is it about.
+
+It helps a lot that the most of the concepts are familiar to us through other languages or libraries we are using. Let's say JavaScript doesn't support currying natively the way Haskell does it, but there are libraries which introduce these concepts. We are very often using [RamdaJS][ramdajs] in our projects, so such things as function composition, currying, partial application and so on were not something totally new to us. They were still had to be "translated" and demonstrated in a Haskell way but we certainly didn't need much time to get the idea.
```

Comment:
Not just “let’s say”, js does indeed not have  currying ;)

Reply:
So concepts such as ... were not completely foreign to us.

Reply:
They were still had to be?

Reply:
Are you talking about the syntax in Haskell which u had to get used to? Syntax and the type strictness? Would be good if u could be more specific

Reply:
> They were still had to be?

I guess this could be simplified to just "They still needed to be".
My intention was to express it like  "sie mussten übersetzt werden" (something like "they had to be translated" then)

Reply:
> Not just “let’s say”

I've meant like "for example". I guess I will change it to that

Reply:
> Are you talking about the syntax in Haskell which u had to get used to? Syntax and the type strictness? Would be good if u could be more specific

That was about things I've mentioned in the previous sentence, like function composition, currying, partial application and so on